MediaBrowser.Controller/Entities/BaseItem.cs

```diff
@@ -2918,5 +2918,27 @@ public virtual long GetRunTimeTicksForPlayState()
         public static readonly IReadOnlyCollection<ExtraType> DisplayExtraTypes = new[] { Model.Entities.ExtraType.BehindTheScenes, Model.Entities.ExtraType.Clip, Model.Entities.ExtraType.DeletedScene, Model.Entities.ExtraType.Interview, Model.Entities.ExtraType.Sample, Model.Entities.ExtraType.Scene };
 
         public virtual bool SupportsExternalTransfer => false;
+
+        public override bool Equals(object obj)
```

Comment:
Add <inheritsdoc/> to these functions.

Reply:
Add what?

Reply:
Maybe `/// <inheritdoc />`?

Reply:
Added

Reply:
Sorry, yes meant to write `/// <inheritdoc/>`